docs/Installation-Script/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Why does this need to be under a separate license? IMO if you're PR-ing it to the StationHub repo, just use the StationHub license.